<ui version="4.0" >
 <class>GaussianBeamWidget</class>
 <widget class="QWidget" name="GaussianBeamWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gaussian Beam</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QToolBox" name="toolBox" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>16</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex" >
        <number>2</number>
       </property>
       <widget class="QWidget" name="page_Cavity" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>248</width>
          <height>487</height>
         </rect>
        </property>
        <attribute name="label" >
         <string>Cavity</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_MagicWaist" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>248</width>
          <height>487</height>
         </rect>
        </property>
        <attribute name="label" >
         <string>Magic waist</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Target waist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_TargetWaist" >
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="value" >
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Target position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_TargetPosition" >
           <property name="minimum" >
            <double>-100000.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="value" >
            <double>550.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QRadioButton" name="radioButton_Tolerance" >
             <property name="text" >
              <string>Tolerance</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_Overlap" >
             <property name="text" >
              <string>Overlap</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_WaistTolerance" >
           <property name="text" >
            <string>Tolerance on waist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_WaistTolerance" >
           <property name="suffix" >
            <string> %</string>
           </property>
           <property name="decimals" >
            <number>1</number>
           </property>
           <property name="minimum" >
            <double>0.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="value" >
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_PositionTolerance" >
           <property name="text" >
            <string>Tolerance on position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_PositionTolerance" >
           <property name="suffix" >
            <string> % Rayleigh</string>
           </property>
           <property name="decimals" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <double>100000.000000000000000</double>
           </property>
           <property name="value" >
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_MinOverlap" >
           <property name="text" >
            <string>Overlap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_MinOverlap" >
           <property name="suffix" >
            <string> %</string>
           </property>
           <property name="maximum" >
            <double>100.000000000000000</double>
           </property>
           <property name="value" >
            <double>98.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_ShowTargetBeam" >
           <property name="text" >
            <string>Show target beam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_OverlapResult" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_MagicWaistResult" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_MagicWaist" >
           <property name="text" >
            <string>Find</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_WaistResult" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Fit" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>248</width>
          <height>487</height>
         </rect>
        </property>
        <attribute name="label" >
         <string>Waist fit</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QComboBox" name="comboBox_FitData" >
           <property name="currentIndex" >
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="fitTable" >
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QPushButton" name="pushButton_FitAddRow" >
             <property name="text" >
              <string>Add row</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_FitRemoveRow" >
             <property name="text" >
              <string>Remove row</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Fit" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Fit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_FitResult" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SetInputBeam" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Set as input beam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SetTargetBeam" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Set as target beam</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
